### DISCUSSION TITLE
paste with bad indent

### DISCUSSION THREAD

**Discussion by rafalpieczyk** [zed-industries/zed#9364] [Feature Requests]
is there a way to copy and paste my code with the same indents? in python I always have to delete a lot of whitespaces to make it like it was intended, with longer classes its a bit annoying, couldnt find a setting to change it

**Comment by lextatic** [comment]
I'm having a similar problem when editing `.yml` files.

If I copy and paste code the inserted code gets some unwanted auto indents.

**Reply by knepley** [reply]
In Atom, you could paste with Shift+Cmd+v and keep the original indentation.

**Comment by choucavalier** [comment]
same for me. right now the pasting in python is pretty annoying as it completely changes the indentation

**Comment by anthony-robin** [comment]
I have the same behavior working on `.slim` files that are indentation based. 
A `Ctrl+Shift+v` shortcut would be a great addition to paste at the cursor position without Zed triggering auto-indentation.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
